circle.yml

```diff
@@ -5,6 +5,8 @@ dependencies:
   pre:
     - sudo apt-get update; sudo apt-get -y install python-pip libglew-dev gdb
     - sudo pip install awscli
+  override:
+    - npm update && npm install
```

Comment:
Correct. The one difference is that `npm update` doesn't run any of the `package.json` scripts (`preinstall`, `postinstall`, `prepublish`).